### PR TITLE
Add sound effects

### LINKED ISSUE #5: Add sound effects

### PR DESCRIPTION
**Pull request by virtosubogdan**
Fixes #5

### CODE REVIEW

File: src/states/endsurvival.ts [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+        this.waves = waves;
         this.setTitle('You have survived ' + waves + ' waves!');
     }
 
     public create(): void {
         super.create();
         this.addOption('Play again', 'survival', null, this.game.world.centerY - 100);
         this.addOption('Return to main menu', 'mainScreen', null, this.game.world.centerY);
+        if (this.waves === 0) {
+            this.game.add.sound(Audio.AudioFemaleWrong.getName()).play();
```
Comment (mariaS210) [inline src/states/endsurvival.ts]: maybe mission failed audio is a better fit here (possibly paired with a mission completed at the end of 3 waves) -> this is open to discussion

File: src/states/game.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import Player from '../components/Player';
 import Enemy from '../components/Enemy';
 import Powerup from '../components/Powerup';
 import TankrApp from '../app';
-import {Images} from '../assets';
+import {Audio, Images} from '../assets';
```
Comment (mariaS210) [inline src/states/game.ts]: it would be nice to have audio for barrel destruction too, since it makes a big kaboom

File: src/components/Enemy.ts [hunk 3]
```diff
@@ -96,6 +98,7 @@ export default class Enemy extends Phaser.Sprite {
 
     public kill(): any {
         super.kill();
+        this.killAudio.play();
```
Comment (mariaS210) [inline src/components/Enemy.ts]: maybe it's better to play the sound on hit function, as kill() is called to destroy the instances on starting a second game. or we make a separate issue to stop using kill() in game create (https://github.com/DankGamesStudios/Tankr/blob/master/src/states/game.ts#L92)

File: src/states/game.ts (hunk 2 above)
Comment (virtosubogdan) [reply src/states/game.ts]: Done

File: src/components/Enemy.ts (hunk 3 above)
Comment (virtosubogdan) [reply src/components/Enemy.ts]: Done

File: src/states/endsurvival.ts (hunk 1 above)
Comment (virtosubogdan) [reply src/states/endsurvival.ts]: Done

**Comment by mariaS210** [comment]
@virtosubogdan Sorry, I thought I merged the Travis integration PR, but i didn't - I merged it just now
If you merge master in this branch then I think Travis status will be reported

**Pull request merged**